### DISCUSSION TITLE
Zooming User Interface  in OS.js

### DISCUSSION THREAD

**Discussion by ghost** [os-js/OS.js#813] [Q&A]
Hi everyone.

I wonder if you've all thought about adding an zooming user interfaces in OS.js. *Have you thought about adding an zooming user interfaces in OS.js?*

#### 1. concept
According to publicly available information on Wikipedia:

> **Note**
> Copyright Wikipedia: *"Zooming user interface or zoomable user interface (ZUI, pronounced zoo-ee) is a [graphical environment](https://en.wikipedia.org/wiki/Graphical_user_interface) where [users](https://en.wikipedia.org/wiki/End-user_(computer_science)) can change the scale of the [viewed area](https://en.wikipedia.org/wiki/Viewport) in order to see more detail or less, and browse through different [documents](https://en.wikipedia.org/wiki/Electronic_document). A ZUI is a type of [graphical user interface](https://en.wikipedia.org/wiki/Graphical_user_interface) (GUI). Information elements appear directly on an infinite [virtual desktop](https://en.wikipedia.org/wiki/Virtual_desktop) (usually created using [vector graphics](https://en.wikipedia.org/wiki/Vector_graphics)), instead of in windows. Users can [pan](https://en.wikipedia.org/wiki/Panning_(camera)) across the virtual surface in two dimensions and [zoom](https://en.wikipedia.org/wiki/Page_zooming) into objects of interest. For example, as you zoom into a text object it may be represented as a small dot, then a thumbnail of a page of text, then a full-sized page and finally a magnified view of the page."*

#### 1.1 Image
<img src="https://raw.githubusercontent.com/andreasbm/web-skills/master/demo.gif" width="60%" />

**image-description:** As we can see here we have a feature to increase or decrease the zoom level of a certain area or interface. Please see this:  [andreasbm/github/web-skills](https://andreasbm.github.io/web-skills/)

#### 2. Idea
I'm thinking of adding the ZUI feature inside os.js. *What do you all think about os.js having zoom support?So... there are things like Figma, Canva that have this feature. you have a zoom area?*

**Comment by ajmeese7** [comment]
Feel free to make a PR in the [client repo](https://github.com/os-js/osjs-client) for review if this is something that you want to see.

**Reply by ghost** [reply]
Hi ajmeese7, thank you for feedback.
